tests/gamma/csm_observability_test.py

```diff
@@ -429,7 +429,10 @@ def query_metrics(
                 view=monitoring_v3.ListTimeSeriesRequest.TimeSeriesView.FULL,
                 retry=retry_settings,
             )
-            time_series = list(response)
+            time_series = []
+            for series in response:
+                if self.is_legit_time_series(series):
+                    time_series.append(series)
```

Comment:
Optional: can be simplified to 
```suggestion
            time_series = list(filter(self.is_legit_time_series, response))
```

Reply:
Done